### DISCUSSION TITLE
Around when Mojo is gonna be almost fully compatible with python

### DISCUSSION THREAD

**Discussion by legitbox** [modular/modular#72] [Q&A] [answered]
or is it even gonna be like a drag and drop solution?

**Comment by TZDak** [comment, marked as answer]
I'm pretty sure it's way too soon to even approximate yet, but I know of two pages on Modular's web site which should be helpful toward that end and should become more helpful in that regard as they get updated.

The first is https://docs.modular.com/mojo/roadmap.html which I'm pretty sure will start to get some approximate future dates when it seems reasonable enough to make such approximations, but so far seems to be focused on what and perhaps in what order but not so much when.

The second is https://docs.modular.com/mojo/changelog.html which actually does have dates but they are all past dates. Those dates and the milestones of progress they mark should give some idea of the pace of progress so far, but that pace may change drastically. If something unexpected or anticipated but difficult gets in the way, the pace may slow substantially, and whether or not that happens there are many ways in which the pace might suddenly increase substantially, but I wouldn't want to try projecting any dates at this stage and I'm guessing neither would those who are more in a position to do so... or that likely would have done so. Hopefully, they will soon.

DAK
